test/index.js

```diff
@@ -1,11 +1,212 @@
+                '</div>';
+            expect(ReactRender.elementToString(element)).to.equal(expectString);
+        });
+
+        it('should render component with array of children correctly', function () {
+            var Item = React.createClass({
+                render: function () {
+                    var props = extend(this.props);
+                    var tag = props.tag;
+                    delete props.tag;
+                    return React.createElement(tag, props);
+                }
+            });
+
+            var Component = React.createClass({
+                render: function () {
+                    return React.createElement(
+                        'div',
+                        {className: 'list'},
+                        this.props.items.map(function (item, i) {
+                            return React.createElement(
+                                Item,
+                                {className: 'item', key: i, tag: 'div'},
+                                'Subtitle' + i,
+                                item.map(function (subitem, j) {
+                                    return React.createElement(Item, {key: j, tag: 'span'}, subitem);
+                                })
+                            );
+                        })
+                    );
+                }
+            });
+            var element = React.createElement(Component, {items: [['a', 'b'], ['c']]});
+            var expectString = '<div class="list">' +
+                '<div class="item">Subtitle0<span>a</span><span>b</span></div>' +
+                '<div class="item">Subtitle1<span>c</span></div>' +
+                '</div>';
+            expect(ReactRender.elementToString(element)).to.equal(expectString);
+        });
+
+        it('should render component with array of children correctly', function () {
+            var Context = React.createClass({
+                getInitialState: function () {
+                    return {
+                        name: 'Aeron'
+                    };
+                },
+                getChildContext: function () {
+                    return {
+                        name: this.state.name
+                    };
+                },
+                render: function () {
+                    return React.createElement('div', null, this.props.children);
+                }
+            });
+
+            var Component = React.createClass({
+                render: function () {
+                    return React.createElement(
+                        'span',
+                        {className: 'name'},
+                        'Name: ' + this.context.name
+                    );
+                }
+            });
+            var element = React.createElement(Context, null, React.createElement(Component));
+            var expectString = '<div><span class="name">Name: Aeron</span></div>';
+            expect(ReactRender.elementToString(element)).to.equal(expectString);
```

Comment:
I think, we need one more test with call like this `ReactRender.elementToString(element, {context: {some data}})`